### ISSUE TITLE
Welcome message does not entirely fit

### ISSUE THREAD

**Issue by jonathan-s** [thepracticaldev/dev.to#2671]
**Describe the bug**
The welcome message does not entirely fit. 

![Screen Shot 2019-05-03 at 11 50 19](https://user-images.githubusercontent.com/2124818/57130471-dc013000-6d99-11e9-832a-4c827c421433.png)

**To Reproduce**

I'm using 1280x800 display. With 100% zoom. You see this the first time you login creating your account. 

**Expected behavior**
You should clearly see let's get started. 

**Desktop (please complete the following information):**

- OS: Macos
- Browser: Firefox.

**Comment by benhalpern** [comment]
Thanks for the heads up.

**Comment by FarazPatankar** [comment]
Taking a look at this now and noticed that the margin we're using is a bit much and that is what is causing the issue. One possible solution would be to use a smaller margin based on the display size but the solution I'd like to go for is to not depend on the margins at all.

We could simply use flexbox with `justify-content` and it will space the items based on the display size automatically. I just tried this implementation and it seems to work fine on all display sizes that I manually tested.

Thoughts? @benhalpern @jessleenyc
